Automattic/wp-calypso
# Update credentials setup message

We're updating the server credentials message to avoid confusion and reduce customers reaching out thinking credentials are needed for Jetpack Backups to work. It's also for consistency with Calypso green where we have the same banner.

#### Proposed Changes

* We changed the message and title in the banner that's shown on the Stats page asking customers to set up their credentials. Now, we're using the same copy that we use in this banner in Jetpack cloud.

#### Testing Instructions



* Buy a Jetpack Backup plan 
* Go to the stats page and check that banner is like in the "After" screenshot

##### Before
<img width="1062" alt="Screenshot 2022-09-16 at 15 57 44" src="https://user-images.githubusercontent.com/298685/190657875-ff141336-63c9-4fae-be2e-675541f1df88.png">

##### After
<img width="1062" alt="Screenshot 2022-09-22 at 17 54 44" src="https://user-images.githubusercontent.com/298685/191800993-9fdf1454-ab60-4400-b907-38cd204d3bae.png">

#### Pre-merge Checklist






Related to #
